docs/serving/samples/hello-world/helloworld-php/index.php

```diff
@@ -1,3 +1,4 @@
 <?php
 $target = getenv('TARGET', true) ?: 'World';
 echo sprintf("Hello %s!\n", $target);
+?>
```

Comment:
```suggestion
?>

```

Adds the missing newline (which also breaks the tests) // cc @maximilien

Reply:
I think it's still mergeable regardless of the new line check failing but thank you!